src/style.css

```
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1), .stars--2 .stars__star:nth-child(-n+2), .stars--3 .stars__star:nth-child(-n+3), .stars--4 .stars__star:nth-child(-n+4), .stars--5 .stars__star:nth-child(-n+5) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1), 
.stars--2 .stars__star:nth-child(-n+2), 
.stars--3 .stars__star:nth-child(-n+3), 
.stars--4 .stars__star:nth-child(-n+4), 
.stars--5 .stars__star:nth-child(-n+5) {
```
Pay attention to the formatting for better readability)